src/Customizer.php

```diff
@@ -51,6 +51,7 @@ public function register() {
 
 		$settings_defaults = apply_filters( 'pt-sticky-menu/settings_default', array(
 			'sticky_selected' => false,
+			'sticky_layout'   => 'up-only',
```

Comment:
@krumpak again, I would replace the "layout" with "visibility"... Don't forget to replace it everywhere else...